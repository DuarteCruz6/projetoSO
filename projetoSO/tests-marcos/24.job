WRITE [(a7,89)]
WRITE [(y7,47)]
BACKUP
WRITE [(f9,99)(w7,46)(x0,18)(q8,86)]
DELETE [q8]
DELETE [x0,w7]
WAIT 3000
WRITE [(n7,93)(t9,98)(e7,52)(j1,37)]
WRITE [(f0,31)]
DELETE [f0]
WRITE [(u9,45)(v1,29)]
READ [f9]
READ [j1]
DELETE [j1,u9]
READ [t9]
WRITE [(o9,25)]
WRITE [(z7,100)(c1,77)(b0,46)]
WRITE [(g8,87)]
WRITE [(z3,33)]
WRITE [(i8,78)]
DELETE [y7,n7,a7,t9]
READ [v1]
DELETE [b0,c1,v1,z7,g8]
WRITE [(p5,4)]
READ [z3]
READ [i8]
WRITE [(v6,91)(t0,90)(h1,70)(q6,87)]
WRITE [(z1,46)(y2,2)(p1,10)(r0,91)]
WRITE [(r6,89)(x0,25)(u5,23)(t1,55)(p0,31)]
WRITE [(o1,5)(e5,0)(p7,46)(w7,98)]
READ [e5]
WRITE [(e9,27)]
SHOW
READ [z3]
WRITE [(v2,2)(q3,9)(x7,90)(b3,16)(w2,50)]
WRITE [(o2,58)(d3,84)(o6,6)(a9,76)(s4,60)]
READ [t1]
WRITE [(f8,29)(k8,51)(u0,68)]
WRITE [(q7,85)]
DELETE [o6,y2,a9,o9]
READ [e5]
SHOW
WRITE [(a4,45)(q4,0)(f2,64)(x2,42)]
WAIT 1000
DELETE [p7,v2,f9,o2,w2]
WRITE [(b7,19)(d6,14)(h3,67)]
WAIT 3000
READ [t0]
WRITE [(d8,39)]
DELETE [e5,t1,u5,i8,v6]
WRITE [(j7,85)(s1,13)(g8,79)(u4,71)(g2,28)]
READ [o1]
WRITE [(z5,97)(n0,90)(i4,10)(b5,61)]
DELETE [a4]
READ [g8]
WRITE [(m0,62)(z8,95)(w2,89)]
WRITE [(m1,70)(i2,14)(c6,47)]
WRITE [(t1,56)(p2,93)]
DELETE [k8]
WRITE [(g6,99)(y3,71)(q0,81)(d2,0)(n4,64)]
WRITE [(b9,77)(y2,9)(b4,66)(l8,10)]
WRITE [(w1,29)(w5,86)(r9,22)(a4,4)]
READ [q6]
WRITE [(c4,60)]
READ [y2]
WRITE [(r1,32)(k6,37)(f3,86)(v2,18)(t5,78)]
WRITE [(j3,14)(v6,61)]
READ [v6]
DELETE [s4,z8,g6,n0,v2]
WRITE [(h9,34)(n0,22)]
WRITE [(c3,15)(t3,76)(r5,24)(d0,47)(d4,76)]
WAIT 2000
WRITE [(p8,96)(w8,72)(r2,15)(b8,40)(l0,27)]
WRITE [(a5,93)]
READ [h1]
WRITE [(o6,60)(h5,84)(u2,45)(p4,97)(j6,89)]
WRITE [(e3,64)]
WRITE [(g3,38)(n5,90)(b0,4)]
WRITE [(e6,30)]
WRITE [(k3,98)(g9,98)]
