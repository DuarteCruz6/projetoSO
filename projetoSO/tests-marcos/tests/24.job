WRITE [(r3,43)(e6,71)(m6,75)(p9,15)]
DELETE [p9,m6]
READ [r3]
DELETE [e6,r3]
WRITE [(x5,37)(w1,84)]
BACKUP
WRITE [(l8,82)(r4,98)(v9,18)(i2,57)]
WRITE [(j9,65)(o3,37)(e1,0)(a7,30)(b1,57)]
READ [x5]
SHOW
DELETE [w1,v9]
SHOW
WRITE [(q6,55)(z2,40)]
DELETE [r4,i2,a7]
WRITE [(y1,68)(b4,63)(x6,25)(s4,96)(y2,71)]
WRITE [(n2,61)]
WRITE [(o9,91)(d1,18)]
READ [x5]
READ [x5]
DELETE [e1,z2,s4,y2,n2]
WRITE [(f9,32)(k8,41)(k6,90)(x3,96)]
SHOW
READ [x5]
WRITE [(d4,7)(e6,8)(a5,84)(s0,72)(m9,58)]
READ [o3]
WRITE [(n4,91)(m4,43)(g6,97)(v0,87)(m4,64)]
WAIT 3000
WAIT 2000
DELETE [j9,q6,x5]
WRITE [(l7,8)(n0,35)(e0,85)(p9,15)]
DELETE [x3,k8,n0,y1]
DELETE [d4,n4,g6,f9,e0]
WRITE [(c1,8)(b3,42)(e9,40)(l0,97)(f7,78)]
WRITE [(i5,4)(w2,35)(h6,90)(h2,28)]
DELETE [l8,m9,w2,k6]
WRITE [(k6,97)(j1,4)(r2,36)(x2,75)(z1,66)]
DELETE [e6]
WRITE [(l3,38)]
WRITE [(t2,36)(c9,56)]
WRITE [(v1,24)(a9,26)]
WRITE [(i8,1)(k4,39)(o4,86)(s4,14)]
WRITE [(p2,9)(z6,80)(i4,24)(w7,52)]
WRITE [(r3,75)]
WRITE [(s8,11)(m8,40)(s8,4)(c5,43)]
DELETE [x6,r3,s0]
READ [i5]
WRITE [(n2,69)(n3,30)(s0,96)(n5,19)]
SHOW
WRITE [(g9,39)]
WRITE [(g3,90)(t5,56)(s3,40)(n6,37)]
READ [s3]
WRITE [(f0,81)(s2,35)(y9,8)]
READ [o9]
DELETE [z6,t2,k4,s8]
WRITE [(u3,100)]
WRITE [(r1,45)(o8,37)(l5,15)(x8,4)]
READ [o3]
WRITE [(g4,52)(n7,9)(x9,27)(u1,70)]
WRITE [(f8,96)(e3,100)(y5,17)]
