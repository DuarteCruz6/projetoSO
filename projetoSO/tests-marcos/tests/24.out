[(r3,43)]
[(x5,37)]
(a7, 30)
(b1, 57)
(e1, 0)
(i2, 57)
(j9, 65)
(l8, 82)
(o3, 37)
(r4, 98)
(v9, 18)
(w1, 84)
(x5, 37)
(a7, 30)
(b1, 57)
(e1, 0)
(i2, 57)
(j9, 65)
(l8, 82)
(o3, 37)
(r4, 98)
(x5, 37)
[(x5,37)]
[(x5,37)]
(b4, 63)
(b1, 57)
(d1, 18)
(f9, 32)
(j9, 65)
(k6, 90)
(k8, 41)
(l8, 82)
(o9, 91)
(o3, 37)
(q6, 55)
(x3, 96)
(x6, 25)
(x5, 37)
(y1, 68)
[(x5,37)]
[(o3,37)]
[(i5,4)]
(a9, 26)
(a5, 84)
(b3, 42)
(b4, 63)
(b1, 57)
(c5, 43)
(c9, 56)
(c1, 8)
(d1, 18)
(e9, 40)
(f7, 78)
(h2, 28)
(h6, 90)
(i4, 24)
(i8, 1)
(i5, 4)
(j1, 4)
(k4, 39)
(k6, 97)
(l3, 38)
(l0, 97)
(l7, 8)
(m8, 40)
(m4, 64)
(n5, 19)
(n3, 30)
(n2, 69)
(o4, 86)
(o9, 91)
(o3, 37)
(p2, 9)
(p9, 15)
(r2, 36)
(s0, 96)
(s8, 4)
(s4, 14)
(t2, 36)
(v1, 24)
(v0, 87)
(w7, 52)
(x2, 75)
(z6, 80)
(z1, 66)
[(s3,40)]
[(o9,91)]
[(o3,37)]
